tests/core/test_feature.py

```diff
@@ -214,3 +214,26 @@ def test_feature_from_dict(dict_data):
         dict_data, field=ln.Feature.name, str_as_cat=False
     )
     assert len(features_with_field) == len(dict_data)
+
+
+def test_feature_from_dict_type(dict_data):
+    # feature_type str
+    features = ln.Feature.from_dict(
+        dict_data, str_as_cat=True, feature_type="Testdata_str"
+    )
+    ln.save(features)
```

Comment:
This can be simplified: Because you `return SQLRecordList(features)` you can do `features.save()`.

Hence, this becomes one line:
```python
features = ln.Feature.from_dict(
    dict_data, str_as_cat=True, feature_type="Testdata_str"
).save()
```